packages/opc-header/README.md

```diff
@@ -0,0 +1,83 @@
+# This is opc-header 👋
+
+Web component based on lit-element, it can be used as a other element of HTML example <opc-header> </opc-header>
+
+https://docs.google.com/document/d/1PoxpPsret2njugkAG_YBmlS3ZNkDd0MTY-JZKDbesrQ/edit#heading=h.w85tasqfwpfc
```

Comment:
Do not add any internal docs to GitHub.

You can write documentation for the component in this README instead.